core/trino-main/src/main/java/io/trino/testing/TestingConnectorSession.java

```diff
@@ -211,7 +211,7 @@ public TestingConnectorSession build()
                     traceToken,
                     timeZoneKey,
                     locale,
-                    start.orElse(Instant.now()),
+                    start.orElseGet(Instant::now),
```

Comment:
(X) Object creation, not so trivial but should be fast

Reply:
I assumed it does a system call, but at least on Linux it should not.

On an Arm-based Mac

```
Benchmark                              Mode  Cnt   Score   Error  Units
BenchmarkOptional.orElseGetInstantNow  avgt  200   9.238 ± 0.032  ns/op
BenchmarkOptional.orElseInstantNow     avgt  200  68.998 ± 0.130  ns/op
```